### PR TITLE
Created welcome page for new project

### PR DESCRIPTION
**Pull request by githubShreyas01**
Added `index.html` file for welcome page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	<a href="two.html">Two</a>
+	<a href="three.html">Three</a>
+	<a href="music.html">Music</a>
+</nav>
+</header>
+
+<section>
+<h2> Favorite Foods </h2>
+<ul>
+	<li>Mangoes</li>
```
Comment (AkshayParihar) [inline index.html:24]: Should it be `Mangoes`?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+<h2> Achievements </h2>
+	<p>Progress in this course(100%)<progress value="1"></progress><br/>
+	Progress in the Specialization capstone(20%)<progress value="20"max="100"></progress><br/>
+	Progress in life goals(51%)<progress value="51"max="100"></progress></p>
+</section>
+
+<section>
+<h2> More About Me </h2>
+     <details open>
+	<summary>My Childhood</summary>
```
Comment (AkshayParihar) [inline index.html:41]: is this `summary` tag correct here?

File: index.html (hunk 1 above)
Comment (githubShreyas01) [reply index.html:24]: Yes this is expected

File: index.html (hunk 2 above)
Comment (githubShreyas01) [reply index.html:41]: Yes this is expected

**Review by AkshayParihar — APPROVED** [review]
great!

**Pull request merged**